### PR TITLE
Perl => 5.36.1

### PR DESCRIPTION
**Pull request by satmandu**
Works properly:
- [x] `x86_64`
- [x] `i686`
- [x] `armv7l` 

### Run the following to get this pull request's changes locally for testing.
```
CREW_TESTING_REPO=https://github.com/satmandu/chromebrew.git CREW_TESTING_BRANCH=perl5361 CREW_TESTING=1 crew update
```

### CODE REVIEW

File: packages/perl_extutils_makemaker.rb [hunk 1]
```diff
@@ -3,11 +3,24 @@
...
-  source_url 'https://cpan.metacpan.org/authors/id/B/BI/BINGOS/ExtUtils-MakeMaker-7.64.tar.gz'
-  source_sha256 '4a6ac575815c0413b1f58967043cc9f2e166446b73c687f9bc62b5eaed9464a0'
+  source_url 'https://cpan.metacpan.org/authors/id/B/BI/BINGOS/ExtUtils-MakeMaker-7.70.tar.gz'
+  source_sha256 'f108bd46420d2f00d242825f865b0f68851084924924f92261d684c49e3e7a74'
+
+  binary_url({
+    aarch64: 'file:///usr/local/tmp/packages/perl_extutils_makemaker-7.70-perl5.36-chromeos-armv7l.tar.zst',
+     armv7l: 'file:///usr/local/tmp/packages/perl_extutils_makemaker-7.70-perl5.36-chromeos-armv7l.tar.zst',
+       i686: 'file:///usr/local/tmp/packages/perl_extutils_makemaker-7.70-perl5.36-chromeos-i686.tar.zst',
+     x86_64: 'file:///usr/local/tmp/packages/perl_extutils_makemaker-7.70-perl5.36-chromeos-x86_64.tar.zst'
```
Comment (uberhacker) [inline packages/perl_extutils_makemaker.rb:16]: Local files?

File: packages/perl_locale_gettext.rb [hunk 2]
```diff
@@ -2,26 +2,28 @@
 
 class Perl_locale_gettext < Package
   description 'Locale::gettext - message handling functions'
-  homepage 'https://metacpan.org/pod/Locale::gettext'
+  homepage 'https://search.cpan.org/dist/Locale-gettext/'
```
Comment (uberhacker) [inline packages/perl_locale_gettext.rb:5]: What the homepage change?

File: packages/perl_io_file.rb [hunk 3]
```diff
@@ -3,13 +3,23 @@
 class Perl_io_file < Package
   description 'Supply object methods for filehandles'
   homepage 'https://metacpan.org/pod/IO::File'
-  version '1.50'
+  version '1.51-perl5.36'
   license 'GPL-1+ or Artistic'
   compatibility 'all'
```
Comment (uberhacker) [inline packages/perl_io_file.rb:8]: `compatibility 'x86_64 i686'`?

File: packages/perl_io_file.rb [hunk 4]
```diff
@@ -3,13 +3,23 @@
...
+  source_sha256 '5493ea55998728cd2b7ecb8234c58fb5d5df27098d0f07addca22444d7616ce0'
 
-  depends_on 'perl_file_temp'
+  binary_url({
+    i686: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/perl_io_file/1.51-perl5.36_i686/perl_io_file-1.51-perl5.36-chromeos-i686.tar.zst',
+  x86_64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/perl_io_file/1.51-perl5.36_x86_64/perl_io_file-1.51-perl5.36-chromeos-x86_64.tar.zst'
+  })
+  binary_sha256({
+    i686: 'f58d3cbf2c72867f2f5be09bd64aea5e9e90f772af466b037b8cab1bf51332e6',
+  x86_64: '7669b1f01815221c7bec2606a4552c4ed220de8a66c0741c2c9a4118b07a631f'
```
Comment (uberhacker) [inline packages/perl_io_file.rb:18]: No arm binaries?

File: packages/perl_locale_gettext.rb (hunk 2 above)
Comment (satmandu) [reply packages/perl_locale_gettext.rb:5]: The old homepage does not work.

File: packages/perl_extutils_makemaker.rb (hunk 1 above)
Comment (satmandu) [reply packages/perl_extutils_makemaker.rb]: oops! Fixed.

File: packages/perl_io_file.rb (hunk 3 above)
Comment (satmandu) [reply packages/perl_io_file.rb:8]: Fixed!

File: packages/perl_io_file.rb (hunk 4 above)
Comment (satmandu) [reply packages/perl_io_file.rb]: fixed!

File: packages/perl_file_temp.rb [hunk 5]
```diff
@@ -3,9 +3,9 @@
 class Perl_file_temp < Package
   description 'Return name and handle of a temporary file safely'
   homepage 'https://metacpan.org/pod/File::Temp'
-  version '0.2311'
-  license 'GPL-1+ or Artistic'
+  version '0.2311-perl5.36'
   compatibility 'all'
+  license 'GPL-1+ or Artistic'
```
Comment (uberhacker) [inline packages/perl_file_temp.rb:8]: Why is this line below compatibility?

**Review by uberhacker — APPROVED** [review]

File: packages/perl_file_temp.rb (hunk 5 above)
Comment (uberhacker) [reply packages/perl_file_temp.rb:8]: No big deal but it's not consistent with the other packages.

**Pull request merged**